.gemspec

```diff
@@ -4,7 +4,7 @@ require 'English'
 
```

Comment:
`.gemspec`? Is this meant to be a hidden file?

Reply:
Googling indicates the used naming convention is `.gemspec`

Reply:
Where did Google take you? I checked a few Ruby gems and all of them use `<name>.gemspec`

Reply:
https://guides.rubygems.org/specification-reference/
https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

Both refer to the `.gemspec`, not sure if it could be a scenario but can only see `<name>.gemspec` being useful in multi-gem project? 🤔

Reply:
https://yehudakatz.com/2010/04/02/using-gemspecs-as-intended/

Same website :P